downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md

```diff
@@ -30,37 +30,31 @@ However, if you are using a version below 2201.0.0 (Swan Lake) and if you alread
 
 ## Install Ballerina
 
-If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.
 
 ## Language updates
 
 ### Bug fixes
 
-To view bug fixes, see the [GitHub milestone for 2201.3.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.3.1+label%3AType%2FBug+is%3Aclosed).
+- [https://github.com/ballerina-platform/ballerina-lang/issues/38583](https://github.com/ballerina-platform/ballerina-lang/issues/38583)
+- [https://github.com/ballerina-platform/ballerina-lang/issues/38518](https://github.com/ballerina-platform/ballerina-lang/issues/38518)
+- [https://github.com/ballerina-platform/ballerina-lang/issues/36270](https://github.com/ballerina-platform/ballerina-lang/issues/36270)
 
 ## Standard library updates
 
-### Improvements
-
 ### Bug fixes
 
 To view bug fixes, see the [GitHub milestone for 2201.3.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+milestone%3A2201.3.1+label%3AType%2FBug+is%3Aclosed).
```

Comment:
To make it consistent , shall we list down issues separately for stdlib issues as well?

Reply:
Done.